src/components/settings/index.vue

```diff
@@ -51,6 +65,12 @@
 					@clicked="setOption('eventStarts')">Desktop notification when an event starts</Toggle>
 				<Toggle :active="alwaysOnTop"
 					@clicked="setOption('alwaysOnTop')">Always on top</Toggle>
+				
+				<h1 class="title">Tray settings</h1>
+				<Toggle :active="hideFromTaskbar"
```

Comment:
This toggle doesn't do it's job right now, it's the one thing that's not finsihed